assets/templates/geography-homepage.tmpl

```diff
@@ -0,0 +1,25 @@
+<div class="slate background--gallery">
+    <div class="wrapper">
+        <nav class="breadcrumb">
+            <ol class="breadcrumb__list">
+                <li class="breadcrumb__item">
+                    <a class="breadcrumb__link" href="https://www.ons.gov.uk/">Home</a>
+                </li>
+                <li class="breadcrumb__item">
+                    Geography
+                </li>
+            </ol>
+        </nav>
+    <h1><b>{{.Metadata.Title}}</b></h1>
+    </div>
+
+</div>
+<div class="wrapper">
+    <h2 class="margin-bottom--2"><b>Area types</b></h2>
```

Comment:
Sorry, I didn't pick this up before but we probably want to stay away from using the `<b>` tag and instead apply font-weight as CSS. The reason being, if we want something to be bold because it is of particular importance to the user then you should use `<strong>`, since screenreaders can process this and emphasise the content to the user. In this case we're using it on a heading tag, which already is emphasised because it's processed as a heading of the page, so we probably just want to to be styling changes. Hope that makes sense, happy to chat through if it doesn't/